docs/medical-api/more-info/medical-record-summary.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Medical Record Summary"
+description: "Consolidate Medical Records Into a Single Document"
+---
+
+This guide will teach you how to generate a Medical Record Summary, which condenses all of patients' medical records into a single document.
+
+To do this, Metriport extracts FHIR resources across all available CDA documents for a patient and de-duplicates them, resulting in a set of unique medical records, which are then written into HTML and PDF files, making them extremely easy to navigate.
+
+Here's the title page of an example Medical Record Summary:
+
+<img className="h-100" src="/images/example-medical-record.png" />
+
+## Generate using API
+
+To generate a Medical Record Summary using Metriport API, you need to follow these steps:
+
+1. Start a [Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post). This endpoint allows you to choose [FHIR resources](/medical-api/fhir/overview), date range, and rendering (`PDF` or `HTML`) for your Medical Record Summary. When this query completes, a [webhook](/medical-api/more-info/webhooks) with a JSON payload containing all these resources will be sent to you.
```

Comment:
this isn't accurate - the webhook will contain a link to the MR summary

also, let's point the dev to the specific webhook they can expect, ie https://github.com/metriport/metriport/blob/630179e498d47866e2ee174c3866a1c827d03b6d/docs/medical-api/more-info/webhooks.mdx#L310